model/task_lifecycle.go

```diff
@@ -628,6 +628,34 @@ func doBisectStepback(ctx context.Context, t *task.Task) error {
 	if nextTask == nil {
 		return errors.Errorf("midway task could not be found for tasks '%s' '%s'", s.LastFailingStepbackTaskId, s.LastPassingStepbackTaskId)
 	}
+	// Log our next task to our last passing and last failing.
+	for _, id := range []string{s.LastFailingStepbackTaskId, s.LastPassingStepbackTaskId} {
```

Comment:
I think it'll definitely be necessary for tasks to be able to both point to the next task that they activated in the stepback process, as well as point to the previous task in stepback that activated them.

I'm noticing now that [setting NextStepbackTaskId](https://github.com/evergreen-ci/evergreen/pull/7341/files#diff-4bef865c41d6d16e1056e61806c23fb8bbc896b3739472dd60f0dd463ef5d2b9L641) is only getting set to the ID of the task itself, which is redundant. We should be setting this field for `t` instead.

Given that `NextStepbackTaskId` only exists for auditability & UI purposes, I ultimately think we can simplify things a bit, remove this loop and instead:
1. Introduce a new field to StepbackInfo, call it `PrevStepbackTaskId` and have it point to the previous task in the stepback process
2. Split `SetStepbackInfo` into two functions -- one that just sets `NextStepbackTaskId` for a task, and another that sets the other 3 fields.  Then, [here](https://github.com/evergreen-ci/evergreen/pull/7341/files#diff-4bef865c41d6d16e1056e61806c23fb8bbc896b3739472dd60f0dd463ef5d2b9R668) we should call the former function on the original task, and the latter function on `nextTask`

Reply:
It used to set the next task as it's own id, but now it never gets set since this loop makes a copy and sets the next stepback id inside.

I think adding the two functions makes sense and will make it cleaner. I don't think we need PrevStepbackTaskId. 

If we have task 1 -> 10 (where 10 is the latest task) and we only know 1 passed, 10 failed (leaving 2-9 unknown). Bisect stepback will choose 5, in which both tasks are the previous tasks. While processing 5, it will set the next task id for 1 & 10 to 5. Then If 5 fails, then it will run stepback on 1 -> 5. In which 5 will not have a next stepback id, because it was the latest in the line of stepback. While 1 will have one, because while processing 5; it got set.

Does this make sense? We can also huddle/zoom to talk about it

Reply:
Talked a bit ofline, going to add the prev stepback task not for the algorithm, but for usability in the UI (navigating back and forth between stepback iterations)